packages/orbit-components/src/ListChoice/README.md

```diff
@@ -28,3 +28,5 @@ Table below contains all types of the props available in the ListChoice componen
 | selected    | `boolean`                  | `false` | If `true`, the check box glyph will be checked.                                                                                                 |
 | **title**   | `Translation`              |         | The title of the ListChoice.                                                                                                                    |
 | action      | `React.Node`               |         | Area for action elements, like Button                                                                                                           |
+| role        | `string`                   |         | Role of the ListChoice                                                                                                                          |
```

Comment:
It has a default value, calculated in runtime (button or checkbox). Should we document it here?